test/integration/targets/connection_sshchroot/test_connection.inventory

```diff
@@ -0,0 +1,9 @@
+[sshchroot]
+sshchroot-pipelining    ansible_ssh_pipelining=true
+sshchroot-no-pipelining ansible_ssh_pipelining=false
+[sshchroot:vars]
+ansible_host=localhost
+ansible_connection=sshchroot
+chroot_path=/
+chroot_executable=/usr/sbin/chroot
```

Comment:
The openSUSE images use `/usr/bin/chroot` instead, so you'll need to generate this file from a template and fill in the correct path to the `chroot` binary.

Reply:
Fixed I add extra vars with the `chroot` path by using `which`